Library/Formula/qt.rb

```diff
@@ -25,6 +25,9 @@ class Qt < Formula
     sha256 "c8a0fa819c8012a7cb70e902abb7133fc05235881ce230235d93719c47650c4e"
   end
 
+  # Make `HOMEBREW_PREFIX/lib/qt4/plugins` an additional plugin search path.
+  patch :DATA
```

Comment:
> I think we've got far too much stuff relying on it to ditch it completely. As long as it continues to work without patches I'm happy just reducing functionality (e.g. plugins) rather than killing it completely.

Agreed. The overall qt ecosystem moves with the evolutionary eagerness of a Panda. I think we can expect to have to somehow basically support `qt` for years, even on a simple "it works" rather than "it's full of functional, shiny features" level.

> All the alternatives I could come up with, that do not involve patching the source, are much more complicated or cumbersome and more difficult to get right.

What were the alternatives you considered?